R/experiment.R

```diff
@@ -9,7 +9,9 @@ make_wkdir <- function(dir, model) {
     message(cat("Using default directory name \"", dir,
                 "\" in current directory \"", getwd(), "\".", sep = ""))
   }
-
+  if(!grepl("/", dir))
+    dir <- paste0(getwd(), "/", dir)
+  
```

Comment:
I don't understand why you add this `if`. Can you explain it to me? 
Because if the "/" is missing in a path, automatically the functions `dir.exists`, `file.exists` and `dir.create` will work in the working directory. So, maybe your `if` is redundant, isn't it?

Reply:
dir can be directory name, in that case dir will be created in current directory
dir can be a path + directory name, in that case dir will be created as indicated

what do you think?

Reply:
I agree with you with the idea, and it's already doing it if we input a directory name without "/", like for example `make_wkdir("test", "model_test")` will automatically create a directory in the current directory even without your `if`. 
But if your input `make_wkdir("path/to/test", "model_test")`, it will create a directory into `path/to`. 
So I don't really understand the use of this `if`.

Reply:
I see what you mean. Yes, it is redundant then.

Reply:
OK, I will remove it then. Thank you =)

Reply:
@choisy reply your question to issue #67 is above

Reply:
Thanks.